code/modules/paperwork/paper.dm

```diff
@@ -306,8 +308,11 @@
 		\[cell\] : Adds a cell to a \[row\]. Required even for the first cell. Can be followed by \[field\] to make the cell a field.<br>
 	</BODY></HTML>"}, "window=paper_help")
 
-
 /obj/item/paper/Topic(href, href_list)
+	if(next_write_time > world.time)
+		message_admins("[usr.ckey] may be spamming paper Topic() calls, likely malicious if this message continues repeating")
```

Comment:
My one concern with this is the strain it'll put on admin chats but this should be fine

Reply:
>  spamming paper Topic()